plugins/plugin-svelte/plugin.js

```diff
@@ -13,7 +13,9 @@ module.exports = function plugin(snowpackConfig, pluginOptions = {}) {
 
   let svelteOptions;
   let preprocessOptions;
-  const userSvelteConfigLoc = path.join(process.cwd(), 'svelte.config.js');
+  // Note(drew): __config is for internal testing use; maybe we should make this public at some point?
+  const userSvelteConfigLoc =
+    pluginOptions.__config || path.join(process.cwd(), 'svelte.config.js');
```

Comment:
Added a config option so that the preprocessor could be tested. Should this be public?

Reply:
Yea, not opposed to supporting a simple JSON object (or JS object) via plugin options, in a follow up PR.

Reply:
Cool. Yeah let me think about it (I actually had trouble finding out about `svelte.config.js` being a thing)